crates/query-engine/metadata/src/metadata/database.rs

```diff
@@ -30,7 +30,7 @@ pub struct ScalarType {
     pub description: Option<String>,
     pub aggregate_functions: BTreeMap<models::AggregateFunctionName, AggregateFunction>,
     pub comparison_operators: BTreeMap<models::ComparisonOperatorName, ComparisonOperator>,
-    pub type_representation: Option<TypeRepresentation>,
+    pub type_representation: TypeRepresentation,
```

Comment:
I am starting to think we should just use `ndc_models::TypeRepresentation` here instead of this type which appears to be a complete copy? Feel bugs will lurk in subtle differences between the two, and I don't understand what the indirection buys us.

Reply:
This could be done in a separate PR to be fair, before or after this one.